src/core/ddsc/include/dds/ddsc/dds_public_status.h

```diff
@@ -306,7 +306,7 @@ dds_get_liveliness_lost_status (
  */
 DDS_EXPORT dds_return_t
 dds_get_offered_deadline_missed_status(
-  dds_entity_t writer,
```

Comment:
This really only takes a writer, so I think "writer" is the better choice.

Reply:
Applied and fixed.